cartesian_trajectory_controller/package.xml

```diff
@@ -90,13 +53,9 @@
   <test_depend>trajectory_msgs</test_depend>
   <test_depend>xacro</test_depend>
 
-  <!-- This is important for the export to work -->
```

Comment:
@fmauch Should we improve that commend instead of removing it? I remember that this costed me some time debugging the export functionality.

Reply:
I would be fine with a better comment :-)